libmscore/glissando.cpp

```diff
@@ -370,6 +380,21 @@ void Glissando::write(Xml& xml) const
       if (_showText && !_text.isEmpty())
             xml.tag("text", _text);
       xml.tag("subtype", int(_glissandoType));
+      writeProperty(xml, P_ID::PLAY_GLISSANDO);
+      switch ( glissandoStyle()) {
+          case MScore::GlissandoStyle::BLACK_KEYS:
```

Comment:
Hi Nicolas, writeProperty writes integers into the xml file.  I want the xml representation to not depend on the accidental integer equivalent of the enum.
It would be great if there were a way to set up string equivalents of these enum values for serialization purposes.



> On 13 May 2015, at 11:40, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/glissando.cpp <https://github.com/musescore/MuseScore/pull/1946#discussion_r30215897>:
> 
> > @@ -370,6 +380,21 @@ void Glissando::write(Xml& xml) const
> >        if (_showText && !_text.isEmpty())
> >              xml.tag("text", _text);
> >        xml.tag("subtype", int(_glissandoType));
> > +      writeProperty(xml, P_ID::PLAY_GLISSANDO);
> > +      switch ( glissandoStyle()) {
> > +          case MScore::GlissandoStyle::BLACK_KEYS:
> It's not a good idea to use spaces in the stored value in XML. So "Black Keys" --> "blackKeys". Also, same remarks than previously, why not using writeProperty?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r30215897>.
>